docs/how-to-use-runhouse.rst

```diff
@@ -0,0 +1,285 @@
+Using Runhouse
+==========================
+This page will guide you can use Runhouse to develop and deploy your ML projects. If you have any questions about what is described here, please reach out to
+`hello@run.house <mailto:hello@run.house>`_ or ping us on Discord and we'd be happy to walk you through the details.
```

Comment:
did we register this email? (what about team or info?)

Reply:
I hallucinated this email I guess. I just made it a group, because it feels friendlier than "support@"